### ISSUE TITLE
TreeView fails AXE accessibility check for role="tree" parent and role="treeitem" child elements relationship

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6321]
### Bug report
TreeView fails AXE accessibility check as it has `role="tree"` on its wrapper `<div>` element, which is not the direct parent of the child `<li>` elements with role="treeitem". The `<ul>` element between tree and treeitem elements should have role="none" set to fix the issue.

### Expected/desired behavior
`role="none"` should be set to the root level `<ul>` element.
 
### Environment
* **Kendo UI version:** 2021.1.224
* **Browser:** [all]

**Comment by veselints** [comment]
Please, review also DropDownTree and FileManager for any issues resulting from the changes in question.
